.gitignore

```diff
@@ -5,3 +5,7 @@ target
 .settings
 .clover
 .*
+.idea
```

Comment:
Hmm, could be. I guess we can reconsider when we next add another `.*` file to the repo.